src/climate_data/generate/scenario_daily.py

```diff
@@ -159,10 +161,13 @@ def generate_scenario_daily_main(  # noqa: PLR0912, PLR0915, C901
     year: str | int,
     target_variable: str,
     cmip6_experiment: str,
+    make_N_draws: int = 5,
```

Comment:
What we want to do is parallelize by draw, rather than having our job generate all jobs.